src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -700,6 +707,15 @@ export default defineComponent({
       }
     },
 
+    getChannelThumbnail: async function () {
+      const thisChannel = this.profileSubscriptions.find((profile) => profile.id === this.channelId)
+      this.channelThumbnail = thisChannel?.thumbnail?.replace(/=s\d+/, '=s35')
+
+      if (this.backendPreference === 'invidious' && this.channelThumbnail) {
+        this.channelThumbnail = youtubeImageUrlToInvidious(this.channelThumbnail, this.currentInvidiousInstance)
+      }
```

Comment:
I think it's better to put URL into a variable first, performing ops like `replace`, `youtubeImageUrlToInvidious` then assign the final value to `this.channelThumbnail` for performance